reactor-netty/build.gradle

```diff
@@ -63,14 +67,12 @@ task docsZip(type: Zip) {
 		into 'docs'
 	}
 
-	if (pdfFile.exists()) {
-		if (!isSnapshot || forcePdf) {
-			logger.lifecycle("pdf ${pdfFile} included in docs zip")
-
-			from(pdfFile) {
-				rename { fileName ->
-					"docs/reactor-netty-reference-guide-${project.version}.pdf"
-				}
+	if (!isSnapshot || forcePdf) {
```

Comment:
Is this change related to java version?

Reply:
yes, when building everything **in one shot** using JDK17, the removal of the `if (pdfFile.exists())` condition allows to include any pdf generated by the :docs:antora task. If we keep the `if (pdfFile.exists())` as before, I have observed that the PDF (if generated) is not actually included.

let me explain:

If we retain the `if (pdfFile.exists())` condition as before, this condition is evaluated during the configuration phase, inaccurately assuming the PDF's absence since it hasn't been generated at that stage (the configuration phase precedes the execution phase, as far as my understanding goes). it's the :docs:antora task which generates the PDF, but during the subsequent execution phase after the configuration phase, so the condition has already been assessed as false, preventing its inclusion in the zip.

Eliminating the `if (pdfFile.exists())` condition fixes the problem. It's worth noting that this solution reflects my current understanding, and there might be other alternative approaches.

Reply:
thanks for the explanation